c_cxx/QNN_EP/QnnCpu/README.md

```diff
@@ -0,0 +1,25 @@
+## About
+- Builds the sample compiled against the ONNX Runtime built with support for Qualcomm® Neural Network (NN) (QNN) Software Development Kit (SDK)
+- The sample uses the QNN EP but runs on QnnCPU.dll (QC)
+- The sample can be compiled and run on an Intel/AMD AMD64 device or QC or other ARM64 device. Does not require a NPU (Neural Processing Unit)
+
+## Prerequisites
+- Windows 11
+- Visual Studio 2022
+- OnnxRuntime ARM Build with QNN support
+    - Either pre-compiled download OR 
+    - Compiled from onnxruntime source with --use_qnn and --qnn_home $(QNN_SDK_ROOT) parameters
+        - If running on AMD64 default params are fine. For ARM64 us the --arm64 option as well when building onnxruntime
+        - Python
+        - QNN SDK (Qualcomm® Neural Network (NN) Software Development Kit (SDK)) from Qualcomm CreatePoint 
```

Comment:
change this section back to https://qpm.qualcomm.com/main/tools/details/qualcomm_ai_engine_direct
thanks!

Reply:
As discussed I will add both

Reply:
Removed links and referencing upcoming official doc